templates/taxi/driver_form.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  <h1>{{ object|yesno:"Update,Create" }} Driver</h1>
+  <form action="" method="post" novalidate>
+    {% csrf_token %}
+    {{ form|crispy }}
+
+    <input type="submit" value="Submit" class="btn btn-primary">
+  </form>
+{% endblock %}
```

Comment:
Add empty line

Reply:
Thanks. 
Didn't know this was a requirement for html files too like for python. I have checked all html files and added the last empty space line.